test/test_jit.py

```diff
@@ -470,6 +470,22 @@ def forward(self, x):
         self.assertExportImport(trace, (t,) + tuple(model.parameters()))
         self.assertExpectedONNXGraph(trace)
 
+    @unittest.skipIf(not IS_WINDOWS, "Testing Fuse skipped on windows")
```

Comment:
Can we remove all the other @unittest.skipIf(IS_WINDOWS) from the other fuser tests now?

Reply:
edit: Oh I guess we can't if there are expect tests...